llvm/lib/Target/AMDGPU/SIDefines.h

```diff
@@ -269,15 +269,6 @@ enum OperandType : unsigned {
   OPERAND_KIMM_LAST = OPERAND_KIMM16
 
 };
-
-// Should be in sync with the OperandSemantics defined in SIRegisterInfo.td
-enum OperandSemantics : unsigned {
```

Comment:
IIRC this was introduced to allow us to tell apart BF16 and F16 in the decoder. We no longer need it?

Reply:
We can tell them apart from the operand type, no need for `OperandSemantics`.